save-cloud-charts/save-cloud/templates/s3-service.yaml

```diff
@@ -0,0 +1,26 @@
+{{- if .Values.s3.external -}}
```

Comment:
I don't think we need a service to get access to S3. mysql requires it since we don't have access to its IP outside and create service to connect to mysql from local machine by port forwarding

Reply:
I guess port forwarding in this case might be convenient for debugging

Reply:
we won't be able to debug external service and we can connect directly to this service

Reply:
Yep, I got my problem, will configure it to deal with minio locally